src/Apps/Collections/Components/CollectionsGrid.tsx

```diff
@@ -65,3 +67,9 @@ export class CollectionsGrid extends Component<CollectionsGridProps> {
     )
   }
 }
+
+const LinkWithoutBorder = styled(Link)`
```

Comment:
Wondering if this will be a common issue in other places. Since this focus is not even currently displayed in most places - maybe it's ok to just kill it in Palette for now?

Or maybe add an option - similar to how `noUnderline` is added. Maybe have something like `noFocusOutline`?